core-bundle/src/Security/Voter/ContentModelAccessVoter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Security\Voter;
+
+use Contao\ContentModel;
+use Contao\FrontendUser;
+
+class ContentModelAccessVoter extends AbstractFrontendAccessVoter
```

Comment:
Should these voters be renamed or moved into a subnamespace? 
So that we don’t get a conflict if we later want to add a backend voter for the content model.

Reply:
well the back end does not use models. And the voter votes on the model, regardless of the scope. But you mean it should be a `ContentModelFrontendAccessVoter` ?